docs/blocking-schema.rst

```diff
@@ -0,0 +1,219 @@
+        "Lambda": 30,
+        "bf-len": 2048,
+        "num-hash-funcs": 5,
+        "K": 20,
+        "input-clks": true,
+        "random_state": 0
+      }
+    }
+
+Schema Components
+-----------------
+.. _blocking-schema/type:
+
+type
+~~~~
+String value which describes the blocking method.
+
+================= ================================
+name              detailed description
+================= ================================
+p-sig             Probability Signature blocking method from `Scalable Entity Resolution Using Probabilistic Signatures on Parallel Databases <https://arxiv.org/abs/1712.09691>`_
+lambda-fold       LSH based Lambda Fold Redundant blocking method from `Scalable Entity Resolution Using Probabilistic Signatures on Parallel Databases <https://arxiv.org/abs/1712.09691>`_
+================= ================================
+
+.. _blocking-schema/version:
+
+version
+~~~~~~~
+
+Integer value that indicates the version of blocking schema
+
+.. _blocking-schema/config:
+
+config
+~~~~~~
+
+A dictionary of configuration to use different blocking methods
+Next we will detail the specific configuration for supported blocking methods.
+
+Specific configuration of supported blocking methods can be found here:
+
+- `config of p-sig <blocking-schema/p-sig>`
+- `config of lambda-fold <blocking-schema/lambda-fold>`
+
+.. _blocking-schema/p-sig:
+
+config of p-sig
+~~~~~~~~~~~~~~~
+===================== ============= ==========================
+configuration         type          description
+===================== ============= ==========================
+blocking-features     list[integer] specify which features u
+filter                dictionary    filtering threshold
+blocking-filter       dictionary    type of filter to generate blocks
+signatureSpecs        list of lists signature strategies where each list is a combination of signature strategies
+===================== ============= ==========================
+
+**Filter Configuration**
+
+============= ============ ==================
+configuration type         description
+============= ============ ==================
+type          string       either "ratio" or "count" that represents proportional or absolute filtering
+max           numeric      for ratio, it should be within 0 and 1; for count, it should not exceed the number of records
+============= ============ ==================
+
+
+**Blocking-filter Configuration**
+
+===================== ============ ==================
+configuration         type         description
+===================== ============ ==================
+type                  string       currently we only support "bloom filter"
+number-hash-functions integer      this specifies how many bits will be flipped for each signature
+bf-len                integer      defines the length of blocking filter, for bloom filter usually this is 1024 or 2048
+===================== ============ ==================
+
+*signatureSpecs configurations*
```

Comment:
This is a different style than **Filter Configuration** and **Blocking-filter Configuration**. I suggest using a sub-heading instead of bold/italics. I think you'd be able to use `''''''''''''''''''`